src/pages/help/faq/index.mdx

```diff
@@ -122,6 +122,10 @@ find it on the
 [Other resources](https://carbondesignsystem.com/designing/design-resources/#native-mobile)
 page.
 
+The Carbon Native mobile Figma kit is available on the
+[Other resources](https://carbondesignsystem.com/designing/design-resources/#native-mobile)
+page.
+
```

Comment:
I think the links are repetitive and the content is confusing here. We can say something like it's available but only a Beta version?

Reply:
If we're concerned about updating this content here again. Maybe we just need to state that it's available and people can find out that it's Beta when it directs them to the Figma kit. 

**Remove**
~We are currently working on a Carbon mobile Figma kit. When it's ready, you'll
find it on the
[Other resources](https://carbondesignsystem.com/designing/design-resources/#native-mobile)
page.~

**Keep**
The Carbon Native mobile Figma kit is available on the
[Other resources](https://carbondesignsystem.com/designing/design-resources/#native-mobile)
page.

Reply:
Oh woops. Yes, I did not delete the original sentence. Thanks Thy!